docs/guide/html-and-static-assets.md

```diff
@@ -39,6 +39,9 @@ By default, a Vue CLI app will automatically generate prefetch hints for all Jav
 
 The hints are injected using [@vue/preload-webpack-plugin](https://github.com/vuejs/preload-webpack-plugin) and can be modified / deleted via `chainWebpack` as `config.plugin('prefetch')`.
 
+### Note for multi page setups
+When using a multipage setup, the plugin name above should be changed to match the structure 'prefetch-{pagename}', for example 'prefetch-app'.
```

Comment:
We can use the `::: tip` container here instead of adding another header (https://vuepress.vuejs.org/guide/markdown.html#custom-containers), like:

```
::: tip Note for Multi-Page Setups
When using a multipage setup, the plugin name above should be changed to match the structure 'prefetch-{pagename}', for example 'prefetch-app'.
:::
```

Reply:
No problem, nice idea. I have done this now.